adapters/visx/visx.go

```diff
@@ -62,8 +89,19 @@ func (a *VisxAdapter) MakeBids(internalRequest *openrtb.BidRequest, externalRequ
 
 	for _, sb := range bidResp.SeatBid {
 		for i := range sb.Bid {
+			bid := openrtb.Bid{}
+			bid.ID = internalRequest.ID
+			bid.CrID = fmt.Sprint(sb.Bid[i].UID)
+			bid.ImpID = sb.Bid[i].ImpID
+			bid.Price = sb.Bid[i].Price
+			bid.AdM = sb.Bid[i].AdM
+			bid.W = sb.Bid[i].W
+			bid.H = sb.Bid[i].H
+			bid.ADomain = sb.Bid[i].ADomain
+			bid.DealID = sb.Bid[i].DealID
```

Comment:
Can you please add a test case to validate the DealID is mapped correctly? I think that's missing.

Reply:
Done.